ocs_ci/utility/utils.py

```diff
@@ -1796,3 +1797,34 @@ def skipif_ocs_version(expressions):
         skip_this = skip_this and eval(comparision_str)
     # skip_this will be either True or False after eval
     return skip_this
+
+
+def get_ocs_version_from_tag(tag):
+    """
+    Parse major.minor version from OCS image tag.
+
+    Args:
+        tag (str): version tag from the image
+
+    Returns
+        str: Version in x.y format
+
+    Raises:
+        ValueError: In case tag which we cannot parse passed.
+
+    """
+    try:
+        version = Version(tag.split(':')[1].lstrip("latest-"))
+        return "{major}.{minor}".format(
+            major=version.major, minor=version.minor
+        )
+    except ValueError as ex:
+        if version.endswith("-rc"):
+            # In case of the tag: 4.2-rc we are not able use semantic version.
+            log.info("X.Y-RC build detected")
+            return version.rstrip("-rc")
+        if version.startswith("latest-"):
+            log.info("latest-X.Y build detected")
+            return version.lstrip("latest-")
```

Comment:
Above all can be handled by uisng Version.coerce

e.g:
```
>>> tag = "quay.io/rhceph-dev/ocs-olm-operator:latest-4.2-rc"
>>> Version.coerce(tag.split(':')[1].lstrip("latest-"))
Version('4.2.0-rc')
>>> "{major}.{minor}".format(major=version.major, minor=version.minor)
'4.2'
>>> 
```

Reply:
done